### ISSUE TITLE
[L4D & 2] Death Item Glow

### ISSUE THREAD

**Issue by Unfellowed** [Target5150/MoYu_Server_Stupid_Plugins#60]
Can you add glow to items dropped by dead infected? Thank you!

https://github.com/Target5150/MoYu_Server_Stupid_Plugins/assets/115072518/342ee736-6a8f-468c-98fe-c97f5bbdc5fd

**Comment by jensewe** [comment]
Nah this is not even a part of the game's default behavior, tracking could be difficult and variable depending on how other plugins work.
Adding glow to entities could be and is supposed to be done when and where they are dropped.

**Comment by Unfellowed** [comment]
Yeah I get it, just hope one day you'll add glow to the secondary weapon. 💙
